### PR TITLE
Fix importing of base64 armored string

### PR DESCRIPTION
**Pull request by alejandrohdezma**
# What needs to be fixed?

I added support for reading a key armored text as a `base64` string in #11 

The problem is that I only added it on reading the key (`opengpg.ts`), but not on importing it (`gpg.ts`) which results in an error when trying to import a key using such a string.

# What is included in this PR?

- Fixes it by adding the same behavior to `gpg.ts`' `importKey` method.
- Add a new step in the `ci` workflow that tests that a GPG key can be imported as a base64 armored string.

### CODE REVIEW

**Comment by alejandrohdezma** [comment]
Trying to figure out why it can import it in tests but not in `ci` workflow...

**Comment by crazy-max** [comment]
@alejandrohdezma

> The ci workflow wasn't building the new stage of the action before using it

No, you don't have to do this. Please see https://github.com/crazy-max/ghaction-import-gpg/blob/master/.github/CONTRIBUTING.md#submitting-a-pull-request

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,6 +24,10 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v2
+      - name: Build
+        run:  |
+          npm install
+          npm run build
```
Comment (crazy-max) [inline .github/workflows/ci.yml:30]: Remove this step. See my comment about `npm run pre-checkin`

Comment (alejandrohdezma) [reply .github/workflows/ci.yml]: Done! https://github.com/crazy-max/ghaction-import-gpg/pull/18/commits/1071df476fb67a0ae68207300af479421a5bae79

Sorry, totally miss that step :(

Comment (crazy-max) [reply .github/workflows/ci.yml]: Thanks :)

**Pull request merged**